### ISSUE TITLE
Best settings for Sony AndroidTV?

### ISSUE THREAD

**Issue by mbnn** [koying/SPMC#543]
Hello all,

Been looking around to find what the best settings are for Sony AndroidTV's (5.1) and HDMI connected receiver. Anyone any idea? 

Also when will the Google play app of SPMC get an update to the latest version? :)

Thank you

**Comment by classicjazz** [comment]
I don't own a Sony Android TV.

But, I would try these video settings:
https://github.com/koying/SPMC/wiki/Recommended-video-settings-for-Android-devices-%28other-than-Nvidia-Shield%29

And these audio settings:
https://github.com/koying/SPMC/wiki/Recommended-passthrough-audio-settings-for-SPMC%2C-NVIDIA-Shield-TV-and-AV-Receivers

Since audio originates on your TV, then you need to pay attention to the ARC section. ARC only supports PCM, Dolby Digital, and DTS soundtracks that will pass over the SPDIF output, not Dolby TrueHD and DTS-HD. So you should uncheck those along with E-AC if your receiver doesn't support that.

**Comment by mrmojo666** [comment]
ARC on my Philips tv doesn't support ac-3 passthrough over 384 kbs, so no support for 640 kbit BD Ac3 tracks, same for DTS, ARC doesn't support DTS 1500 Kbs . I think it is not related to philips but to ARC specifications. In my case i need to use Adroid raw passthrough mode to get low bitrates ac3 and dts working.

**Comment by classicjazz** [comment]
Does everything work if you disable passthrough and let SPMC do all decoding in software?

**Comment by mrmojo666** [comment]
yep, but always following ARC limitation PCM 2.0 channels ;)  no multichannel PCM output ;)
